modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -23,12 +24,10 @@ This procedure requires knowledge of the workload utilizing the kernel module an
 
 * Ensure that the user workload (the application running in the cluster that is accessing kernel module) is not running on the node prior to kernel module unloading and that the workload is back running on the node after the new kernel module version has been loaded.
```

Comment:
This seems like two things, one to do before the upgrade, and a verification step for afterwards: 
```suggestion
* Ensure that the user workload (the application running in the cluster that is accessing kernel module) is not running on the node prior to kernel module unloading.

* Ensure that the workload is back running on the node after the new kernel module version has been loaded.
```